### DISCUSSION TITLE
部署一个公共的接口

### DISCUSSION THREAD

**Discussion by abigmiu** [ikaros-dev/ikaros#227] [General]
如果我想提交admin 的pr。 但是得首先部署一个 java 服务。 这样有点麻烦。能不能用服务器部署一个公共的接口。可以让人访问，测试，开发

**Comment by chivehao** [comment]
> 如果我想提交admin 的pr。 但是得首先部署一个 java 服务。 这样有点麻烦。能不能用服务器部署一个公共的接口。可以让人访问，测试，开发

这个问题我们有考虑过，类似halo的 https://demo.halo.run  ，但是由于Ikaros很多剧集资源都是大文件，如果部署一个公共的DEMO服务，没有服务器带宽跟的上的。考虑到这点，我们暂时还没有部署DEMO演示站。

**Comment by chivehao** [comment]
> 如果我想提交admin 的pr。 但是得首先部署一个 java 服务。 这样有点麻烦。能不能用服务器部署一个公共的接口。可以让人访问，测试，开发

如果想要提交admin的pr，因为目前基本没啥文档，欢迎随时联系我们提供支持
